### DISCUSSION TITLE
Logging

### DISCUSSION THREAD

**Discussion by DotNetDublin** [BlazingSaddlesSolutions/TimeKeeper#24] [General]
I believe the main choices for logging within dot net applications is currently [NLog](https://nlog-project.org/) and [Serilog](https://serilog.net/) although I'd welcome any other suggestions.

Regardless of the logging platform implemented I would like the project to demonstrate when and what to log.

**Comment by harperjohn** [comment]
Yes - Logging is not "glamorous"  but it is extremely useful! (if not "essential")

**Comment by AnkurSheel** [comment]
Do we care about error reporting?

**Comment by DotNetDublin** [comment]
> Do we care about error reporting?

That would be an excellent area to explore. Any suggestions?

**Comment by AnkurSheel** [comment]
I have used https://sentry.io/welcome/ before. It's fine but there might be better alternatives out there

**Reply by DotNetDublin** [reply]
I've created a new discussion to examine error reporting.
